components/tabs/tabs.js

```diff
@@ -722,7 +722,9 @@ class Tabs extends LocalizeStaticMixin(ArrowKeysMixin(RtlMixin(LitElement))) {
 		this._tabInfos.forEach((tabInfo) => {
 			if (tabInfo.selected && tabInfo.id !== selectedTab.controlsPanel) {
 				tabInfo.selected = false;
-				this._getPanel(tabInfo.id).selected = false;
+				const panel = this._getPanel(tabInfo.id);
+				// panel may not exist if it's being removed
+				if (panel) panel.selected = false;
```

Comment:
When a panel is removed, its corresponding tab removal is animated, and so it's tabinfo still exists in `this._tabinfos`... so when we set the select status of panels we cannot assume the panel exists.